roles/rhsm/tasks/4_enable_repo.yml

```diff
@@ -0,0 +1,12 @@
+#Enabling the repos
+- name: Enable Repos | Enabling the satellite repos
+  command: >-
```

Comment:
why not `rhsm_repository` module?

Reply:
Could implement the module in a role with something like:

```
  rhsm_repository:
    name:  "{{ item.name }}"
    state: "{{ item.state | default('present') }}"
    purge: "{{ item.purge | default('no') }}"
  loop: "{{ repos }}"
```

And then make the direct call to that role with something like:

```
         - role: rhsm-repository
           repos:
            - rhel-{{ ansible_distribution_major_version }}-server-satellite-{{ satellite_version }}-rpms
            - rhel-{{ ansible_distribution_major_version }}-server-rpms
```

Reply:
added the rhsm_repository module to the play